### PR TITLE
Remove Grafana, Rollbar, Papertrail

### PR DESCRIPTION
**Pull request by chimedpalden**
## Description
This PR removes Gragana, Rollbar, Papertrail. Only Datadog will be used for log management.

## Database schema changes
N/A

## Tests
N/A

### Manual test cases run
N/A

### CODE REVIEW

File: test/spec/logger/logger.spec.ts
Comment (jjalan) [inline test/spec/logger/logger.spec.ts:45]: Do we have a spec for datadog?

**Review by jjalan — APPROVED** [review]
LGTM.

Please approve once @bbbneo333 approves

**Review by bbbneo333 — APPROVED** [review]

**Pull request merged**
